templates/mas_config.yaml.j2

```diff
@@ -46,3 +47,18 @@ templates:
   translations_path: /mas/share/translations
 policy:
   wasm_module: /mas/share/policy.wasm
+{% if smtp_configuration is not none %}
+email:
+  from: '"The almighty auth service" <auth@example.com>'
```

Comment:
Could this be replaced by something like MAS [homeserver value] ?

Reply:
I updated the email template